tools/resources/__init__.py

```diff
@@ -269,7 +269,7 @@ def get_file_refs(self, file_type):
                 if not any(
                    path.startswith(dirname(e.path)) for e in self._excluded_libs
                 ):
-        	    to_ret.append(ref)
+                    to_ret.append(ref)
```

Comment:
I switched editors recently, until it decided to never insert spaces when I type `<tab>`. I have switch back because that's infuriating when working with python.